src/data/weather.ts

```diff
@@ -342,7 +352,8 @@ export const getWeatherStateSVG = (state: string): SVGTemplateResult => {
 
 export const getWeatherStateIcon = (
   state: string,
-  element: HTMLElement
+  element: HTMLElement,
+  nighttime?: boolean
```

Comment:
Lets make this camel case
```suggestion
  nightTime?: boolean
```